### PR TITLE
Feature/rpnv1 704 exponential pacemaker

### PR DESCRIPTION
**Pull request by clstrfsck**
Initial implementation of an exponential backoff pacemaker.

### CODE REVIEW

**Comment by clstrfsck** [comment]
Note that this is targeted onto the infrastructure branch, so there is some possibility of reviewing in a sensible way.  Before merging, I'll retarget against the rc branch.

I've left it as a draft for now to remind me to change the branch before merging.

File: radixdlt/src/main/java/com/radixdlt/EpochsConsensusModule.java [hunk 1]
```diff
@@ -57,10 +57,14 @@
  */
 public class EpochsConsensusModule extends AbstractModule {
 	private static final int ROTATING_WEIGHTED_LEADERS_CACHE_SIZE = 10;
-	private final int pacemakerTimeout;
+	private final long pacemakerTimeout;
+	private final double pacemakerRate;
+	private final int pacemakerMaxExponent;
 
-	public EpochsConsensusModule(int pacemakerTimeout) {
+	public EpochsConsensusModule(long pacemakerTimeout, double pacemakerRate, int pacemakerMaxExponent) {
```
Comment (talekhinezh) [inline radixdlt/src/main/java/com/radixdlt/EpochsConsensusModule.java:64]: Maybe time to start using a module for configuration parameters like pacemaker timeout? But not a big deal.

**Review by siy — APPROVED** [review]
LGTM

**Review by talekhinezh — APPROVED** [review]

File: radixdlt/src/main/java/com/radixdlt/EpochsConsensusModule.java (hunk 1 above)
Comment (clstrfsck) [reply radixdlt/src/main/java/com/radixdlt/EpochsConsensusModule.java:64]: Yeah, I was planning on moving the pacemaker into its own module as part of the bigger changes coming up, so this should tidy this up quite a bit.

**Pull request merged**